modules/govuk/manifests/deploy/setup.pp

```diff
@@ -24,27 +29,37 @@
     $aws_ses_smtp_password,
     $gemstash_server = 'http://gemstash.cluster',
     $ssh_keys = { 'not set in hiera' => 'NONE_IN_HIERA' },
+    $deploy_uid = undef,
+    $deploy_gui = undef,
```

Comment:
`deploy_gid`?

Reply:
Did you see this typo, @afda16? It looks like you need to replace `$deploy_gui` with `$deploy_gid`.